File: cdf-155A.c
#include<stdio.h>
int main(){
    int a;
    scanf("%d",&a);
    int b;
    scanf("%d",&b);d
    int max=b,min=b,count=0;
    for(int i=1;i<a;i++){
        scanf("%d",&b);
        if(b>max){
            count++;
            max=b;
        }else if(b<min){
            count++;
            min=b;
        }
    }
    printf("%d\n",count);
    return 0;

}

File: cdf-155A.py
n=int(input())
x=list(map(int,input().split()))
max=x[0]
min=x[0]
count=0
for i in range(1,n):
    if(x[i]>max):
      count+=1
      max=x[i]
    if(x[i]<min):
       count+=1
       min=x[i]

print(count)
